specification/templates/.github/CODEOWNERS

```diff
@@ -1,5 +0,0 @@
-# Learn about CODEOWNERS file format:
-#  https://help.github.com/en/articles/about-code-owners
-#
-# Global Owners
-* @signalfx/TEAM-maintainers
```

Comment:
Should change this to mention approvers team, not delete

Reply:
Fair